.github/workflows/hil.yml

```diff
@@ -29,18 +29,9 @@ env:
 
 jobs:
   build-xtasks:
-    name: Build xtasks | ${{ matrix.host.arch }}
+    name: Build xtasks
     runs-on: ubuntu-latest
```

Comment:
It was not worth to use the macos self hosted runner here?

Also, we can already use `ubuntu-24.04-arm`here to avoid cross compiling one of the binaries. Not sure how much time we will save though.

Reply:
We can certainly try an ARM runner, but the difference will be seconds. This workflow isn't a bottleneck, but the mac runner can be if we throw more and more on it.

Reply:
Well that didn't work out :) I can try fiddling with non-cross builds but maybe not in this one.